src/lib/crypto.cpp

```diff
@@ -160,6 +160,11 @@ pgp_generate_seckey(const rnp_keygen_crypto_params_t *crypto,
     }
     seckey->sec_protection.s2k.usage = PGP_S2KU_NONE;
     seckey->material.secret = true;
+    /* validate generated key material */
+    if (validate_pgp_key_material(&seckey->material, rng)) {
```

Comment:
I think it was agreed we can drop this if I understood correctly.

Reply:
@dewyatt Yeah, will remove this code and update PR shortly.